tools/.golangci.yaml

```diff
@@ -24,6 +24,7 @@ linters:
     - stylecheck
     - unused
     - unconvert # Remove unnecessary type conversions
+    - unparam
```

Comment:
Note, this is new check that was not enabled before, so this is not part of https://github.com/etcd-io/etcd/issues/16610 and should start from community discussion whether we want to adopt `unparam` check at all.